### ISSUE TITLE
search: the scrollbar and scroller are too narrow, almost completely unusable 

### ISSUE THREAD

**Issue by grahamperrin** [zhukov/webogram#1382]
![telegram web](https://cloud.githubusercontent.com/assets/192271/24590014/b704e73e-17dc-11e7-87f0-7be5f23a93cd.png)

Ridiculously difficult to target when working with, for example, a trackball. 

Environment: Firefox 52.0.1 (64-bit) on FreeBSD-based TrueOS Desktop.

**Comment by zhukov** [comment]
Yes, currently the app requires having trackpad / mouse with scroll wheel for comfortable use.

**Comment by azzimuth** [comment]
@zhukov this causes problems using stickers as the list is scrolled too fast and some stickers are not available if one scrolls using a mouse wheel. One has to drag the scrollbar but, since it's tiny, that's not easy either.

I will file a separate issue on problems with the list of stickers, if none is already present.

**Comment by zhukov** [comment]
@grahamperrin did you try TDesktop app? It has a bit wider scrollbar. Is it wide enough for you?

**Comment by grahamperrin** [comment]
Thanks for the idea … 

> FreeBSD-based

– re: https://github.com/zhukov/webogram/blob/master/README.md#maintained-locations is anything there compatible?
